src/app/furniture-store-modal/furniture-store-modal.component.less

```diff
@@ -1,19 +1,24 @@
 @import (reference) "../../styles.less";
 
-.inventoryContainer{
+.furnitureModal {
+  overflow: hidden;
+}
+
+.inventoryContainer {
   position: relative;
   height: 100%;
 }
 
 .itemSlot {
-  width: 90px;
-  height: 90px;
+  width: 75px;
+  height: 75px;
   background-color: var(--storeButtonColor);
+  line-height: 12px;
 }
 
 .priceLabel {
-width: 60px;
-text-align: center;
+  width: 60px;
+  text-align: center;
 }

```

Comment:
Is this CSS being utilized any longer?

Reply:
Same reasoning as the left over html, I'll remove it and push it today.